Swabbr.Core/Services/VlogService.cs

```diff
@@ -113,20 +104,22 @@ public async Task LikeAsync(Guid vlogId, Guid userId)
             userId.ThrowIfNullOrEmpty();
 
             var vlogLikeId = new VlogLikeId { VlogId = vlogId, UserId = userId };
-            using (var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled))
+            using var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
+
+            if (await _vlogLikeRepository.ExistsAsync(vlogLikeId).ConfigureAwait(false)) { throw new OperationAlreadyExecutedException("User already liked given vlog"); }
```

Comment:
Remove all ConfigureAwait in .NET Core / .NET 5.0

Reply:
Will do so in separate PR